server/handlers.go

```diff
@@ -518,6 +527,15 @@ func getCurrentApplication(r *http.Request) *models.Application {
 	return nil
 }
 
+func getTarget(a *models.Application, t string) (string, error) {
```

Comment:
Like I said in the other comment: I think making the signature of these functions `getTarget(a *models.Application, t *models.Target)` would buy us more in the long run.

Also: how about renaming the function to `findTarget` so it matches the `findApplication` function just below this one?